src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -531,6 +531,7 @@
       <MonoAOTCompilerDefaultAotArguments Include="deterministic" />
       <MonoAOTCompilerDefaultAotArguments Include="mattr=simd" Condition="'$(WasmSIMD)' == 'true'" />
       <MonoAOTCompilerDefaultProcessArguments Include="--wasm-exceptions" Condition="'$(WasmExceptionHandling)' == 'true'" />
+      <MonoAOTCompilerDefaultProcessArguments Include="--wasm-gc-safepoints" Condition="'$(WasmEnableThreads)' == 'true' or '$(WasmEnablePerfTracing)' == 'true'" />
```

Comment:
I'm not sure if this will work.

Actually what do we want this to do:
* pick up the flag from how `build.sh --os browser` was running?
* or pick up the property from what the user app build is doing?

I think right now it's from whatever the user passes when doing the app build.
I think that's probably what we want.